### DISCUSSION TITLE
Arixv2Md

### DISCUSSION THREAD

**Discussion by charleson98** [InternLM/Tutorial#2242] [project]
Arixv2Md是一个自动从arixv下载论文，利用LLM抽取摘要和关键图片的工具。计划逐渐打造成一站式论文收集整理hub

# v1.0.0目标实现
1、huggingface daily paper拉取
2、arixv论文latex下载、解析
3、论文摘要中文翻译改写
4、论文图片选取

**Comment by invisible30** [comment]
你好，了解一下
